pkg/jaeger/store/get_operations.go

```diff
@@ -34,7 +35,7 @@ WHERE
 
 func getOperations(ctx context.Context, conn pgxconn.PgxConn, query spanstore.OperationQueryParameters) ([]spanstore.Operation, error) {
 	var (
-		pgOperationNames, pgSpanKinds pgtype.TextArray
+		pgOperationNames, pgSpanKinds pgtype.FlatArray[pgtype.Text]
```

Comment:
I wonder why we use PG array instead of `[]string`? My understanding is that PG arrays should only be used if we need to access to some of it's internals.

Reply:
The same question applies to other trace functions where we use PG array and just convert to string slice. I got a bit confused. Maybe I am missing something?

Reply:
I do understand that array was used prior to this change so it's not directly part of this refactoring, but just wanted to get understanding and maybe fix it if not needed.

Reply:
>IIRC it's due to the way data is serialised/deserialised, PGX expect one of its types because of the interfaces it implements. I'll double check this.

Any updates on this one? Thinking if we can be using `[]string`